.goreleaser.yaml

```diff
@@ -9,7 +9,7 @@ builds:
       - goos: darwin
         goarch: 386
     flags:
-      - -tags={{if eq .Env.MAINNET "true"}}mainnet{{else}}testnet{{end}}
+      - -tags=netgo,ledger,{{if eq .Env.MAINNET "true"}}mainnet{{else}}testnet{{end}}
```

Comment:
To stay consistent with our other templates lines, put in `"`?

Reply:
This was here first and... I kind of hate the `\"` :joy: